README.md

```diff
@@ -115,3 +115,13 @@ conflict resolution. See below list with the proposed solution:
   two step work. First move I2CCLK_MAX and I2CCLK_MIN marco from i2c source
   file to header file. Then split the I2CCLK_MIN for each supported transfer
   mode.
+
+- gd32l23x
+
+  - `SystemInit` function in `system_gd32l23x.c` contain an invalid usage of
+    `nvic_vector_table_set` function, just remove it.
+
+  - `gd32l23x_libopt.h` file not exist, add it manually.
```

Comment:
Is gd32l23x_libopt.h really necessary since we have [hal_gigadevice](https://github.com/zephyrproject-rtos/hal_gigadevice)/[common_include](https://github.com/zephyrproject-rtos/hal_gigadevice/tree/main/common_include)?

Reply:
The `gd32l23x_libopt.h` is used at HAL internal, for `system_gd32l23x.c` and other peripheral source like `gd32l23x_usart.c`

Reply:
Ok, using internally is not a problem if there is no reference in *.h files.